src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2464,16 +2464,16 @@ namespace NServiceBus.Sagas
     }
     public class SagaFinderDefinition
     {
-        public SagaFinderDefinition(System.Type type, string messageType, System.Collections.Generic.Dictionary<string, object> properties) { }
-        public string MessageType { get; }
+        public System.Type MessageType { get; }
+        public string MessageTypeName { get; }
```

Comment:
Cant follow the full discussion here since I am on my mobile right now, but
-1 for internal ctors on public types when there is not a very good reason
to do so. Makes testing potentially so much harder.
Am 23.05.2016 22:10 schrieb "David Boike" <notifications@github.com>:

> In
> src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt
> <https://github.com/Particular/NServiceBus/pull/3756#discussion_r64279999>
> :
>
> > @@ -2464,16 +2464,16 @@ namespace NServiceBus.Sagas
> >      }
> >      public class SagaFinderDefinition
> >      {
> > -        public SagaFinderDefinition(System.Type type, string messageType, System.Collections.Generic.Dictionary<string, object> properties) { }
> > -        public string MessageType { get; }
> > +        public System.Type MessageType { get; }
> > +        public string MessageTypeName { get; }
>
> Yes, its value is MessageType.FullName.
>
> Would you rather have it called MessageTypeFullName? I think an actual
> property is required to remove any ambiguity about what "name" is supposed
> to be used for comparisons to message headers, etc.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3756/files/639ea3a32ddd6b358458eca279cdbb7e101002cc#r64279999>
>